pranama-ui/src/components/View/services/login_service/LoginForm.vue

```diff
@@ -98,17 +98,7 @@
   </div>
 </template>
 <script>
-import { API } from "@/shared/index.js";
-import axios from "axios";
-
-const apiClient = axios.create({
-  baseURL: `${API}`,
-  withCredentials: false,
-  headers: {
-    Accept: "application/json",
-    "Content-Type": "application/json"
-  }
-});
+import ContactService from "@/services/api/ContactService";
 
 export default {
```

Comment:
axios.js should go to the shared folder ( which is good ) . Other API services calls can you move to a folder apiservices under src. ie src/apiservices.

Reply:
fixed in 24e2b49